modules/skybrowser/src/wwtcommunicator.cpp

```diff
@@ -290,16 +291,34 @@ glm::dvec2 WwtCommunicator::equatorialAim() const {
 }
 
 void WwtCommunicator::setImageOrder(int i, int order) {
+    using it = std::deque<std::pair<int, double>>::iterator;
     // Find in selected images list
-    auto current = findSelectedImage(i);
-    auto target = _selectedImages.begin() + order;
+    it current = findSelectedImage(i);
```

Comment:
Can also just do `auto it current = findSelectedImage(i);`  we allow `auto` in place of container iterators